docs/examples/sudoers.d/fty_00_base.in

```diff
@@ -34,10 +34,10 @@ Cmnd_Alias BIOS_LOGHOST_RSYSLOG   = @libexecdir@/@PACKAGE@/loghost-rsyslog
 Cmnd_Alias BIOS_MYSQLDUMP   = /usr/bin/mysqldump
 Cmnd_Alias BIOS_DMF         = /usr/bin/bios-dmf
 Cmnd_Alias BIN_SYSTEMCTL    = /bin/systemctl
-Cmnd_Alias BIN_IP_ROUTE     = /bin/ip route
+Cmnd_Alias BIN_FTY_ROUTE    = @libexecdir@/@PACKAGE@/fty-route
```

Comment:
Not sure we need this at all - the script does `sudo` for commands that need elevation and are already allowed.